### PR TITLE
Adds blog to navigation

### PR DESCRIPTION
**Pull request by superhighfives**
image

Waiting on the URL for the navigation, but this PR adds a link to blog to the header.

### CODE REVIEW

File: src/config.toml [hunk 1]
```diff
@@ -16,4 +16,10 @@ theme = "buildpacks"
   identifier = "docs"
   name       = "Docs"
   url        = "/docs"
-  weight     = 0
+  weight     = 2
+
+  [[menu.main]]
+  identifier = "blog"
+  name       = "Blog"
+  url        = "https://medium.com/"
```
Comment (jkutner) [inline src/config.toml]: Do we need to wait to merge this until we have a proper URL, or can we merge it now?

Comment (superhighfives) [reply src/config.toml]: Hey @jkutner. Need to wait until we have a proper URL—have you created the publication in Medium?

Comment (hone) [reply src/config.toml]: I updated this to point to the buildpacks medium i created.

**Comment by hone** [comment]
@superhighfives @jkutner should we wait until the medium is fully setup before merging and deploying?

**Comment by superhighfives** [comment]
Yep, I'd say let's hold off until we have some content in there, @hone @jkutner. 🙌

**Comment by sclevine** [comment]
Rebased against master and added commits to publish it. Will merge before 9am tomorrow morning.

**Pull request merged**